### PR TITLE
feat: replace ribbon with GitHub corner

### LINKED ISSUE #16: GitHub ribbon can be replaced with newer GitHub Corner

### PR DESCRIPTION
**Pull request by hamirmahal**
closes #16.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -232,10 +233,34 @@
...
-  <a href="https://github.com/llimllib/bloomfilter-tutorial"><img
-      style="position: absolute; top: 0; left: 0; border: 0;"
-      src="https://github.blog/wp-content/uploads/2008/12/forkme_left_red_aa0000.png?resize=149%2C149"
-      alt="Fork me on GitHub"></a>
+  <a
+    aria-label="View source on GitHub"
+    class="github-corner"
+    href="https://github.com/llimllib/bloomfilter-tutorial"
+    rel="noopener"
+    target="_blank"
```
Comment (hamirmahal) [inline index.html:241]: This PR changes the link behavior so that it opens in a new tab.

**Comment by hamirmahal** [comment]
# Without this change

[Screencast from 10-09-2024 12:03:07 PM.webm](https://github.com/user-attachments/assets/e5518e14-0901-4a21-8d95-419c8ab36847)

# With this change

[Screencast from 10-09-2024 12:03:39 PM.webm](https://github.com/user-attachments/assets/8b86e5d6-d820-4f3c-b9be-29d885064f7b)

**Comment by llimllib** [comment]
Why open it in a new tab? I feel like a user could command-click on it, or right click -> open in new tab if that's what they want?

**Comment by hamirmahal** [comment]
That's fair. I changed the behavior because my preference is to have links open in a new tab by default, instead of shifting the user's focus.

Do you want me to revert that behavior?

**Comment by llimllib** [comment]
Thank you! It's live now: https://llimllib.github.io/bloomfilter-tutorial/

**Comment by hamirmahal** [comment]
You're welcome! I'm happy to help.

**Pull request merged**